tools/flake8_plugin.py

```diff
@@ -61,6 +82,28 @@ def visit_Name(self, node: ast.Name) -> None:
 
         self.generic_visit(node)
 
+    def visit_Call(self, node: ast.Call) -> None:
+        name_list: List[str] = []
+        func = node.func
+        while isinstance(func, ast.Attribute):
+            name_list.append(func.attr)
+            func = func.value
+        if not isinstance(func, ast.Name):
+            return
```

Comment:
this seems over complicated -- this should probably inspect the expected attributes exactly?